docs/source/user_guide/runtime-conf.md

```diff
@@ -166,9 +166,10 @@ Example: `['test','unsecured']`
 This section defines the settings for the Kubeflow Pipelines deployment that you want to associate with this runtime configuration.
 
 ##### api_endpoint
+
 The KubeFlow Pipelines API endpoint you want to utilize. This setting is required.
 
-Example: `https://kubernetes-service.ibm.com/pipeline`
+KubeFlow Example: `https://kubernetes-service.ibm.com/pipeline`
```

Comment:
I don't think we need to call out that this particular example is Kubeflow given we're clearly in the Kubeflow Settings section.  We also don't prefix `Example` in other examples.
```suggestion
Example: `https://kubernetes-service.ibm.com/pipeline`
```

Reply:
done